code/_onclick/item_attack.dm

```diff
@@ -31,19 +31,27 @@ avoid code duplication. This includes items that may sometimes act as a standard
 
 // No comment
 /atom/proc/attackby(obj/item/W, mob/user, var/click_params)
+	user.client.interact_log += "([time_stamp()]) - Interacted with [src], using [W]."
```

Comment:
I suggest adding the `user`'s current intent as well.

Reply:
Good idea